css/css-fonts/font-palette-vs-shorthand.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Fonts Module Level 4: interaction of font-palette and font shorthand</title>
+<link rel="help" href="https://drafts.csswg.org/css-fonts/#font-prop">
+<meta name="assert" content="font-palette is reset to normal by font shorthand.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style id="style">
+@font-face {
+    font-family: colr;
+    src: url("resources/COLR-palettes-test-font.ttf") format("truetype");
+}
+div {
+    margin: 10px;
+}
+#a {
+    font: 50px colr;
+    font-palette: dark;
+}
+#b {
+    font-palette: dark;
+    font: 50px colr;  /* should reset font-palette to normal */
+}
+#c {
+    font-palette: dark;
+    font-size: 50px;
+    font-family: colr;
+}
+#d {
+    font-palette: dark;
+    font-size: 50px;
+    font-family: colr;
+    font-palette: normal;
+}
+</style>
+</head>
+<body>
+<p>The first and third examples should use the 'dark' palette; the second and fourth, 'normal'.</p>
+<div id=a>A</div>
+<div id=b>A</div>
+<div id=c>A</div>
+<div id=d>A</div>
+<script>
+test(function() {
+    let testElem = document.getElementById("a");
+    let computed = window.getComputedStyle(testElem);
+    assert_equals(computed.fontPalette, "dark");
+    assert_equals(computed.font, "");
```

Comment:
Did you want to change these lines where font is "", if yes, please file a separate issue with the question about the serialization of font shorthand when font-palette is non-initiail.

Reply:
Yes, I will file a separate issue for that, thanks!